### PR TITLE
docs: removed folder access step

### PR DESCRIPTION
**Pull request by UlisesGascon**
## Main change

I am using `ansible [core 2.13.6]` with `MacOS 12.4 (21F79)` and the execution from the `ansible` folder does not include the plugins. It is working fine from root. 

### Screenshots

Captura de Pantalla 2022-11-18 a las 21 47 47

Captura de Pantalla 2022-11-18 a las 21 46 44

### CODE REVIEW

**Review by richardlau — APPROVED** [review]

**Review by AshCripps — APPROVED** [review]
I made this change a while ago and thought I had caught all the doc. Good catch!

File: doc/ssh.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ for build working group members.
     ```
 4. Follow the instructions in the [ansible guide](../ansible/README.md) to
    install ansible on your local machine.
-5. Run `ansible-playbook playbooks/write-ssh-config.yml` from the `ansible/`
-   directory in the root of this repo, then the host information will be
+5. Run `ansible-playbook playbooks/write-ssh-config.yml` from the root of this repo,
```
Comment (targos) [inline doc/ssh.md:30]:
```suggestion
5. Run `ansible-playbook ansible/playbooks/write-ssh-config.yml` from the root of this repo, 
```

Comment (UlisesGascon) [reply doc/ssh.md]: Great catch @targos !

**Comment by UlisesGascon** [comment]
CI is fixed in #3086

**Pull request merged**